packages/patch/patch.1.0.0/opam

```diff
@@ -8,7 +8,7 @@ bug-reports: "https://github.com/hannesm/patch/issues"
 license: "ISC"
 
 depends: [
-  "ocaml" {>= "4.04.2"}
+  ("ocaml" {>= "4.04.2" & < "5.0"} | "ocaml" {>= "5.0"} & "base-bytes")
   "dune"
```

Comment:
Should fix the lowerbounds jobs: 1.4.0 includes https://github.com/ocaml/dune/pull/1391
```suggestion
  "dune" {>= "1.4.0"}
```
(the addition of the lowerbound is independent of the ocaml 5 support, obviously)